test/integration/kpatch-test

```diff
@@ -117,14 +117,7 @@ log() {
 }
 
 unload_all() {
-	for i in `/sbin/lsmod |egrep '^kpatch' |awk '{print $1}'`; do
-		if [[ $i != kpatch ]]; then
-			$KPATCH unload $i >> $LOG 2>&1 || error "\"kpatch unload $i\" failed"
-		fi
-	done
-	if /sbin/lsmod |egrep -q '^kpatch'; then
-		$RMMOD kpatch >> $LOG 2>&1 || error "\"rmmod kpatch\" failed"
-	fi
+	$KPATCH unload --all
```

Comment:
The script change looks fine, but I wanted to add a nitpick on the commit message:
"The unload_all() function in kpatch-test script is out dated and does not unload patched..."  that last word should be "patches".

Reply:
Fixed in new version, thanks.